types/lodash/scripts/generate-all.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 # If some one want to update lodash modules, you must run this script under the same dir where it lives.
 # Or you can run them(generate-modules.ts) separately by hand
+# This script may be deprecated. See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/41857#issuecomment-580417935
```

Comment:
in that case maybe we should just delete it.

Reply:
Right.

Reply:
I was not sure if it was truly OK to delete the file because I haven't been fully familiar with the development workflow of @types/lodash and while @aj-r taught me about `generate-all.sh` in https://github.com/DefinitelyTyped/DefinitelyTyped/issues/41857#issuecomment-580417935, it was not 100% decisive...
but yes, I should have deleted it.
It's one reason we are using VCS :)